fragments/rhn-register.sh

```diff
@@ -44,7 +85,7 @@ subscription-manager repos \
                      --enable="rhel-7-server-rpms" \
                      --enable="rhel-7-server-extras-rpms" \
                      --enable="rhel-7-server-optional-rpms" \
-                     --enable="rhel-7-server-ose-3.2-rpms"
+                     --enable="rhel-7-server-ose-${OSE_VERSION}-rpms"
```

Comment:
Nope, I see bellow in this patch that OSE_VERSION is propagated by heat's "str_replace" function - this means that heat does s/$OSE_VERSION/<value>/ before passing the script to the instance (As part of cloud-init user data).